src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -539,14 +539,14 @@ private List<SyntaxTrivia> GetPermutedTrivia(CSharpSyntaxNode node, List<XmlElem
 
                             return convertedType == symbol.ContainingType;
                         })
-                .Select(n => semanticModel.GetSymbolInfo(n, cancellationToken).Symbol);
+                .SelectAsArray(n => semanticModel.GetSymbolInfo(n, cancellationToken).Symbol);
 
-            return convertedMethodGroups.Select(symbolAndProjectId.WithSymbol);
+            return convertedMethodGroups.SelectAsArray(symbolAndProjectId.WithSymbol);
```

Comment:
This is actually worse isn't it, as you have to realize all the intermediate arrays?